src/cljs/main/broadfcui/page/method_repo/method/details.cljs

```diff
@@ -68,15 +71,19 @@
             {:text "Yes"
              :onClick #(mc-sync/flag-synchronization)
              :href (nav/get-link :workspace-method-config (:dest-workspace-id @state) (:dest-config-id @state))}}])
-        [:div {:style {:display "flex" :marginTop "1.5rem" :padding "0 1.5rem"}}
+        [:div {:style {:display "flex" :marginTop "1.5rem" :padding "0 1.5rem" :alignItems "flex-end"}}
          (tab-bar/render-title
           "METHOD"
           [:span {}
            [:span {:data-test-id "header-namespace"} (:namespace method-id)]
            "/"
            [:span {:data-test-id "header-name"} (:name method-id)]])
-         [:div {:style {:paddingLeft "2rem" :marginTop -3}}
+         [:div {:style {:marginLeft "2rem" :marginTop -3}}
           (this :-render-snapshot-selector)]
+         (when (:is-public @state)
+           [:span {:style {:alignSelf "center" :marginLeft "2.5rem"}}
+            (icons/render-icon {} :public)
```

Comment:
put a bit of a margin-right on the icon, rather than starting the text with a space.

Reply:
Thanks, the space felt kinda dumb